main-app/client/src/theme.js

```diff
@@ -0,0 +1,62 @@
+import { createMuiTheme } from '@material-ui/core/styles'
+
+function pxToRem(value) {
+  return `${value / 16}rem`
+}
+
+const theme = createMuiTheme({
+  palette: {
+    white: { main: '#FFFFFF' }, //  Button Text, Data Row Background, Top Navigation
+    dataRowBackgroundAlt: { main: '#F4F6F8' },
+    primaryText: { main: '#1F2933' },
```

Comment:
Remove line 11 & add the color to the existing theme variables "palette.text.primary"
The theme object is located here https://material-ui.com/customization/default-theme/